src/plugin/plugin.rs

```diff
@@ -38,87 +46,125 @@ impl<PhysicsHooksData> RapierPhysicsPlugin<PhysicsHooksData> {
     pub fn pixels_per_meter(pixels_per_meter: f32) -> Self {
         Self {
             physics_scale: pixels_per_meter,
+            system_setup: true,
             _phantom: PhantomData,
         }
     }
+
+    /// Provided for use when staging systems outside of this plugin using
+    /// [`with_system_setup(false)`](Self::with_system_setup) enabled.
+    /// See [`PhysicsStages::SyncBackend`] for a description of these systems.
+    pub fn get_sync_backend_systems(&self) -> SystemSet {
+        // TODO: Error if not skipped?
```

Comment:
What is the best way to ensure invalid uses of systems are discouraged?

Reply:
Potential for error handling based on the plugin configuration.  Otherwise, if no error handling is needed, this method could simply be an export outside of any `impl`.

Reply:
I would keep it under `impl` and remove `&self`. This way you don't need any error handling and functions will be as a part of the plugin impl struct.